### DISCUSSION TITLE
Set Address to false after short wait if it got set to true

### DISCUSSION THREAD

**Discussion by shanse01** [Supergiovane/node-red-contrib-knx-ultimate#369] [Q&A] [answered]
Dear all,

It is probably a very stupid question but all I want to do is set an address to false after a short wait once it got set to true before.

I keep ending up in getting the a "Node disabled due to a circular reference" error.

Is there any simple solution to this?

Thanks for your kind support and regards
Sven

**Comment by Supergiovane** [comment, marked as answer]
Hi
Just remove the msg.topic
If you paste here your flow, i’ll fix it, but you’ll find all infos in the wiki, just search for “circular”
